js/NetworkPane.js

```diff
@@ -0,0 +1,221 @@
+import React from 'react';
+
+const Pane = require('./Pane');
+
+class NetworkPane extends React.Component {
+  _paneRef = null;
+  _networkRef = null;
+  _width = null;
+  _height = null;
+
+  handleDownload = () => {
+    var link = document.createElement('a');
+    link.download = `${this.props.title || 'visdom_image'}.jpg`;
+    link.href = this.props.content.src;
+    link.click();
+  };
+
+  CreateNetwork = graph => {
+    var width = 500,
+      height = 500;
```

Comment:
width and height attributes should be extracted from the `opts` dict, shouldn't they?

Reply:
Hi @dizcza , Actually there is a error in it which i forgot to mention, Actually the problem is networkPane is getting props.height and props.width as Null. So to remove it temporarily I have given a 500 value to them. It would be nice if someone can check why it is happening. 
Thanks

Reply:
I am just looking at the code, and I have noticed that height and width are undefined in PlotPane also. I also explored the possibility of using this.props.w and this.props.h but they are very small (24, 30) in my case. so is there any conversion that I am missing?

Reply:
Sadly, I cannot comment on this issue. Perhaps, @JackUrb could clarify this.

Reply:
Indeed `w` and `h` are scaled, in this case to be able to adapt to the `ReactGridLayout` cells rather than px sizes. These are computed for the layout of the current added pane here: https://github.com/fossasia/visdom/blob/master/js/main.js#L371-L373